backend/src/handlers/helpful_links_handler.go

```diff
@@ -81,6 +82,30 @@ func (srv *Server) handleAddHelpfulLink(w http.ResponseWriter, r *http.Request,
 	return writeJsonResponse(w, http.StatusCreated, "Link added successfully")
 }
 
+func (srv *Server) handleSubmitAllowlistRequest(w http.ResponseWriter, r *http.Request, log sLog) error {
+	linkID, err := strconv.Atoi(r.PathValue("id"))
+	if err != nil {
+		return newInvalidIdServiceError(err, "Invalid id")
+	}
+	link, err := srv.Db.GetLinkFromId(uint(linkID))
+	if err != nil {
+		return newDatabaseServiceError(err)
+	}
+	log.add("link", link.Url)
+	cliams := r.Context().Value(ClaimsKey).(*Claims)
+	user, err := srv.Db.GetUserByID(cliams.UserID)
+	if err != nil {
+		return newDatabaseServiceError(err)
+	}
+	emailReq := newAllowlistRequest(user, link)
```

Comment:
Is this sending the email to the user (the Admin requesting access) or is it sending it to a different user?

Reply:
nvm.